tests/templates/plaintext.eml

```diff
@@ -0,0 +1,14 @@
+Message-ID: <3deb436d-348f-06ba-2e34-93f751cc3aa2@student.dhbw-mannheim.de>
```

Comment:
Always the same message ID? I don't think thats a good idea. But since we currently don't send the MessageID, I dont think it matters. But should be kept in mind

Reply:
Good point. The message id is currently being sent with the payload and zimbraweb does not send the same message twice.
One the one hand this is a good thing since we're now just passing the message id through (to allow for threading and stuff) but for testing this means that the emails don't actually get delivered. I will fix this.